### DISCUSSION TITLE
Project Status

### DISCUSSION THREAD

**Discussion by gaby** [FiloSottile/mkcert#444] [General]
What's the status of this project? There's dozens of good PR's that never get reviewed/approved. There hasn't been any changes to the tool in almost a year.

**Comment by FiloSottile** [comment]
_o/ I generally work on projects in batches, unless there's something urgent. I keep an eye on all issues and PRs to catch urgent stuff or to form an idea of what changes should go into the next batch. mkcert has a specific job, which is to make certificates that work in browsers locally. As far as I can tell it's doing that job well, am I missing serious blockers?

**Comment by gaby** [comment]
@FiloSottile Thanks for the update, really appreciate it. 

Quick good to have fixes #344 #379 #309 and #274

PR #327 will be critical coming Ubuntu 22.04 which will install Firefox as a Snap app.

Another one would be #359 although tests are failing with 0 changes in code.

**Comment by FiloSottile** [comment]
Ah, good to know about Ubuntu 22.04, thank you. I will look into #327 sooner rather than later.

The rest look like good-to-have improvements, which is the kind of thing that will go into the next batch of mkcert work, when I manage to get around to it.

#309 is a much bigger UX change than the others, BTW.

**Comment by gaby** [comment]
Thank you! Yeah agree, most are good to have. #327 will defenitely be important coming Ubuntu 22.04 in April, 2022.

**Comment by potatoqualitee** [comment]
Not sure if this is in scope for browser-intent but I can no longer use mkcert in my CI/CD process on macOS, and  https://github.com/FiloSottile/mkcert/issues/415 is a blocker for me, as my command line browsers don't trust the certs.

**Comment by FiloSottile** [comment]
I've gotten around to #327 and some low-hanging fruit. #415 would be nice to fix but is not easy, so I won't block a release on it. https://github.com/FiloSottile/mkcert/pull/309 and https://github.com/FiloSottile/mkcert/pull/274 I am still thinking about.

**Reply by wnunezc** [reply]
It will be possible to obtain the 2022-23-24 updates that have been proposed in the PRs, you have abandoned this project for 2 years, I think there are people waiting, or mark the project as discontinued...

**Reply by gaby** [reply]
Rather than archiving, ideally it gets transferred to someone else or an Org for maintenance. It's up to @FiloSottile though
